docs/ml_tasks.md

```diff
@@ -56,7 +47,7 @@ For prediction, Substra creates a new docker container containing the algorithm
 
 Substra then runs the algorithm **in testing mode** and predictions are saved to the **pred** volume.
 
-At this point, the score computation step starts and works exactly like the one in training. 
+At this point, the score computation step starts and works exactly like the one in training.
```

Comment:
This is not true anymore.

Reply:
Good catch! I updated this line with the wording I removed from the training section above.